package.json

```diff
@@ -12,7 +12,7 @@
     "url": "https://github.com/cypress-io/cypress-realworld-app/issues"
   },
   "engines": {
-    "node": "^16.16.0"
+    "node": "^18.15.0"
```

Comment:
Depending on any app update, it may be possible to include also Node.js `20` and later e.g.

`"node": "^18.15.0 || >=20.0.0"`